apps/examples/testcase/le_tc/compression/tc_compress_read.c

```diff
@@ -0,0 +1,88 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <debug.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <tinyara/config.h>
```

Comment:
Please place the header file like below sequence.
1. tinyara/config.h
2. standard header : stdio.h, unistd.h, string.h and so on.
3. tinyara/xxx.h
4. absolute path header : "xxx.h"

Reply:
Changes made